bundles/core/src/main/java/com/adobe/cq/commerce/core/components/models/button/Button.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019 Adobe.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.adobe.cq.commerce.core.components.models.button;
+
+import org.osgi.annotation.versioning.ProviderType;
+
+@ProviderType
+public interface Button {
```

Comment:
This should not be needed at all if we use `com.adobe.cq.wcm.core.components.models.Button`.